src/libraries/System.DirectoryServices.Protocols/tests/System.DirectoryServices.Protocols.Tests.csproj

```
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetFrameworkCurrent)</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-Linux;$(NetCoreAppCurrent)-Windows_NT;$(NetFrameworkCurrent)</TargetFrameworks>
```

Can/should this be

```suggestion
    <TargetFrameworks>$(NetCoreAppCurrent);$(NetFrameworkCurrent)</TargetFrameworks>
```

Unless there are split compilations in the tests, that should work.

---

If we do that now then it would also run on netcoreapp osx which we don't yet currently support. I do intend adding that in a subsequent PR, and when I do I will be updating the configurations here, but for now we do need to keep the rid-specific to avoid running on osx.